src/diy/README.md

```diff
@@ -40,6 +40,6 @@ Download the firmware from our github, define what boards you have put in your t
 
 **5. Install and set up the SlimeVR Server**
 
-This final step requires you to install and configure your server, allowing the tracking data to be parsed on your local machine. After doing so, all that's left is to configure tracker roles and you are ready to go! The steps to do all this can be found on our [SlimeVR Server setup page](../server-setup/slimevr-setup.md).
+This final step requires you to install and configure your server, allowing the tracking data to be parsed on your local machine. After doing so, all that's left is to configure tracker roles and you are ready to go! The steps to do all this can be found on our [SlimeVR Server setup page](../server/index.html).
```

Comment:
```suggestion
This final step requires you to install and configure your server, allowing the tracking data to be parsed on your local machine. After doing so, all that's left is to configure tracker roles and you are ready to go! The steps to do all this can be found on our [SlimeVR Server setup page](/server/index.html).
```